challengeutils/helpers.py

```diff
@@ -58,3 +58,35 @@ def create_team_wikis(syn, synid, templateid, tracker_table_synid):
             #syn.sendMessage(i[])
             #Store copied synId to tracking table
             syn.store(synapseclient.Table(tracker_table_synid,[[wiki_copy[templateid],i['teamId']]]))
+
+
+def kill_docker_submission_over_quota(syn, evaluation_id, quota=None):
```

Comment:
this only works with the workflow hook though?

Reply:
Yes. This only works with the hook.